edxval/api.py

```diff
@@ -104,6 +105,35 @@ def create_video(video_data):
         raise ValCannotCreateError(serializer.errors)
 
 
+def update_video(video_data):
+    """
+    Called on to update Video objects in the database
+
+    update_video is used to update Video objects.
+    Args:
+        video_data (dict):
+         {
+                url: api url to the video
+                edx_video_id: ID of the video
+                duration: Length of video in seconds
+                client_video_id: client ID of video
+                encoded_video: a list of EncodedVideo dicts
+                    url: url of the video
+                    file_size: size of the video in bytes
+                    profile: ID of the profile
+                courses: Courses associated with this video
+         }
+    """
+
+    video = _get_video(video_data.get("edx_video_id"))
```

Comment:
@BenjiLee ```_get_video``` already returns ```ValVideoNotFoundError``` so I would say keep it DRY. But can update doc that it returns ```ValVideoNotFoundError``` from ```_get_video```